lib/matplotlib/axes/_axes.py

```diff
@@ -3974,72 +3974,47 @@ def bxp(self, bxpstats, positions=None, widths=None, vert=True,
 
         zdelta = 0.1
 
-        def line_props_with_rcdefaults(subkey, explicit, zdelta=0,
-                                       use_marker=True):
+        def merge_kw_and_rc(subkey, explicit, zdelta=0, marker=True):
```

Comment:
I prefer keeping the name `use_marker` to indicate that this is a bool, even if that means wrapping lines in one or two places. Otherwise, we'd have to introduce a docstring for `merge_kw_and_rc()` IMHO.

Reply:
With `merge_kw_rc` and `usemarker` things just fit :-)